addons/mail/static/src/messaging.js

```diff
@@ -5,6 +5,7 @@ import { deserializeDateTime } from "@web/core/l10n/dates";
 import { Deferred } from "@web/core/utils/concurrency";
 import { url } from "@web/core/utils/urls";
 import { htmlToTextContentInline, removeFromArray } from "./utils";
+import { generateBody } from "@mail/mail_message";
```

Comment:
I would rename function `prettifyMessageContent` or something like that.
Also filename should be named to something like `@mail/message_utils`.

Reply:
While I agree that mail_message is probably not the best name for this file, I also think utils is almost always a bad name since it just mean "everything".
Might as well call this file message_prettify.js, at least it doesn't mean you can put everything inside.

Reply:
ok